src/humanloop/requests/tool_kernel_request.py

```diff
@@ -1,6 +1,5 @@
 # This file was auto-generated by Fern from our API Definition.
 
-import typing_extensions
```

Comment:
looks like a lot of edits in autogenerated files - don't think we should bother; has no functional difference and it's just noisy - maybe flag to fern that there are duplicate imports and they might look into cleaning it up

Reply:
I pinged fern on the internal slack channel